### PR TITLE
Auto-generate multus conf. Do not exit sriov DS

### PR DESCRIPTION
**Pull request by krsna1729**
Instead of relying on configmap to be created/customized by the user to
generate multus conf, we autogenerate it using the current default CNI
conf file on the host. This way CNI's with host specific conf details
will continue to work.

```
{
  "type": "calico",
  "nodename": "clr-01",
   ...
}
```

Since SR-IOV device may not be present on all nodes, the daemonset
should not exit as this will put the pod in restart loop. This would
cause the kubelet and runtimes to repeatedly restart, as per
initContainer logic.

Signed-off-by: Saikrishna Edupuganti <saikrishna.edupuganti@intel.com>

### CODE REVIEW

File: clr-k8s-examples/9-multi-network/multus-sriov-ds.yaml [hunk 1]
```diff
@@ -87,6 +87,20 @@ data:
         user: multus
     current-context: multus-context
     EOF
+  install-multus-conf.sh: |
```
Comment (mcastelino) [inline clr-k8s-examples/9-multi-network/multus-sriov-ds.yaml]: @krsna1729 is this cleaned up? What will happen the next time this script runs? Will it not see 00-multus.conf at /host/etc/cni/net.d

Comment (krsna1729) [reply clr-k8s-examples/9-multi-network/multus-sriov-ds.yaml]: @mcastelino  Ooh thats right. Good catch.

**Comment by krsna1729** [comment]
@mcastelino addressed.

**Comment by mcastelino** [comment]
LGTM

**Pull request merged**